README.md

```diff
@@ -144,6 +144,10 @@ _Apply for $100-$3,000 fast microgrants to pursue your effort in longevity. You
 https://xorgrant.org <br>
 _Xor Grant is a no-strings-attached microgrant awarded to high school seniors pursuing a computer science or cybersecurity program in college. Its mission is to support students who are passionate about solving interesting problems and love thinking outside the box._
 
+## Bellare Grants
+https://bellaregrants.netlify.app <br>
+Bellare Grants is a program supporting local/Indian initiatives in education, healthcare, research, and open source projects(hardware and software) through financial contributions.
```

Comment:
```suggestion
Bellare Grants is a program supporting local/Indian initiatives in education, healthcare, research, and open source projects (hardware and software) through financial contributions.
```

space seems to be missing

Reply:
@matkoniecz Updated here https://github.com/nayafia/microgrants/pull/84/commits/16041fd930fc75da635341be1f7262ee2c103324